privval/sc_file_test.go

```diff
@@ -0,0 +1,103 @@
+package privval
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/BlockscapeNetwork/signctrl/config"
+	"github.com/BlockscapeNetwork/signctrl/types"
+	"github.com/stretchr/testify/assert"
+	tm_crypto "github.com/tendermint/tendermint/crypto"
+	tm_ed25519 "github.com/tendermint/tendermint/crypto/ed25519"
+	tm_privval "github.com/tendermint/tendermint/privval"
+	tm_prototypes "github.com/tendermint/tendermint/proto/tendermint/types"
+	tm_types "github.com/tendermint/tendermint/types"
+)
+
+type TestFilePV struct{}
```

Comment:
This struct is never used in this  file. Is it needed for other files or can it be removed

Reply:
It's used in `request_handler_test.go`, so it is needed.